apiserver/Makefile

```diff
@@ -1,35 +1,9 @@
-.PHONY: install-hail-locally build push run run-hail rm deploy build-hail-shadow-jar
+.PHONY: install-hail-locally build push run run-hail rm deploy build-hail-shadow-jar \
+	build-spark-base build-hail-base build-spark-master build-spark-worker build-apiserver
```

Comment:
You should put the targets that are only defined in the conditional in a .PHONY statement inside the conditional.